services/accumulo-service/src/main/java/datawave/microservice/accumulo/config/AccumuloProperties.java

```diff
@@ -0,0 +1,44 @@
+package datawave.microservice.accumulo.config;
```

Comment:
I wonder if this should go in a different module (accumulo-utils? or the service starter). It seems a number of services might need access to an accumulo connection and it would be nice to not re-create the same class over and over (it's already in audit for the accumulo auditor).

Reply:
Yep, had been thinking the same thing, just didn't want to stir the pot too much until I knew this was closer to being fully baked ;) Same with the AuditClient...I packaged that stuff separately, knowing this shouldn't really be it's final home